### PR TITLE
chore: Migrate to Androidx

### LINKED ISSUE #1: Update README
The current README file is intentionally funny because it was meant to be displayed at I/O Extended Maputo '18. But since the event has already been held, the README can now have a more clean and explanatory format.

I would suggest using this [README template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2).

### LINKED ISSUE #9: Migrate to AndroidX
Most of the [AndroidX](https://developer.android.com/jetpack/androidx) packages are now stable. I believe it's time we update our app to use AndroidX. See the [migration guide here](https://developer.android.com/jetpack/androidx/migrate).

### PR DESCRIPTION
**Pull request by EdNgulele**
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar).
- [x] I have read the [Contribution Guide](CONTRIBUTING.md) and my PR follows them.
- [x] I have discussed these changes via issue.
- [x] I have made my changes on a new branch.
- [x] My code follows the code style of this project.

### Short description of what this PR does:
- Migrates support libraries to androidx
- Upgrades firebase dependencies to their latest stable versions

closes #9

### CODE REVIEW

**Comment by rosariopfernandes** [comment]
[GitHub experienced an incident](https://blog.github.com/2018-10-21-october21-incident-report/) and background services are currently disabled, therefore the automatic build on Travis CI wasn't triggered. I'll review once that issue gets fixed.

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (EdNgulele) [inline app/src/main/res/layout/activity_main.xml]: I'm sorry, i just noticed that I changed the layout to ConstraintLayout.

Comment (rosariopfernandes) [reply app/src/main/res/layout/activity_main.xml]: But does it still work as expected?

Comment (rosariopfernandes) [reply app/src/main/res/layout/activity_main.xml]: Just saw it. I have changed it back to CoordinatorLayout.

**Comment by rosariopfernandes** [comment]
Thanks @EdNgulele !

**Pull request merged**